pallets/validators-manager/src/tests/mock.rs

```diff
@@ -405,6 +405,7 @@ parameter_types! {
     pub const ErasPerGrowthPeriod: u32 = 2;
     pub const RewardPaymentDelay: u32 = 2;
     pub const RewardPotId: PalletId = PalletId(*b"av/vamgr");
+    pub const MaxCandidates: u32 = 257;
```

Comment:
why 257?

Reply:
because of maximum_collators_is_reached limit, so that the tests don't clash with the MaxCandidates

Reply:
I'm out of the loop here, but I'd like to hear the continuation here :grin: 
It feels like this hints at some hidden dependency between variables / constants in different places. It would be good if the code could make them visible somehow, instead of using specific magic numbers without explanation.

Reply:
this is irrelevant, after the refactor of the `maybe_inserted` variables in the `lib.rs` the number of these candidates in the `validators-manager` pallet is absolutely inconsequential